cluster-autoscaler/core/scaleup/orchestrator/orchestrator_test.go

```diff
@@ -124,6 +124,223 @@ func TestMixedScaleUp(t *testing.T) {
 	simpleScaleUpTest(t, config, expectedResults)
 }
 
+func TestAtomicScaleUpOK(t *testing.T) {
```

Comment:
can we merge tests introduced in this file into one, table driven test?

Reply:
Done